emails/tests/mgmt_process_emails_from_sqs_tests.py

```diff
@@ -447,7 +446,8 @@ def test_ses_slow(
     assert rec2_extra["success"] is True
     assert rec2_extra["message_process_time_s"] < 120.0
     assert rec2_extra["subprocess_setup_time_s"] == 1.0
-    assert mock_process_pool_future._timeouts == [1.0]
+    # future.wait(1.0) was called 3 times, was ready after the 3rd call.
```

Comment:
praise: nice comment. now I see how this correlates to the 3-item `_is_stalled.side_effect` list.